src/Microsoft.ML.Data/Transforms/TransformBase.cs

```diff
@@ -388,7 +390,7 @@ public VectorType GetSlotType(int col)
                 return new Bindings(parent, infos, inputSchema, false, names);
             }
 
-            public void Save(ModelSaveContext ctx)
+            internal void Save(ModelSaveContext ctx)
```

Comment:
>internal [](start = 12, length = 8)

it's in private class, any reason for this change?

Reply:
I guess everything should be as conservative as possible?

---
In reply to: [255247956](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255247956) [](ancestors = 255247956)

Reply:
Yes you are right I can undo this

---
In reply to: [255253748](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255253748) [](ancestors = 255253748,255247956)